patternfly/patternfly-elements
# Set up files array for packages to control compiled assets

## Set up files array for packages to control compiled assets

### What has changed and why
This branch sets up our build tasks to check the package.json for the files array object as a means of controlling which assets are added to our root directory.

### Testing instructions

1. `npm run demo`
2. Ensure all components continue to work (no regressions)
3. Check compiled assets in directories to ensure they match the designated files array from the package.json (if one exists)

Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
